src/components/Button/styles.module.scss

```diff
@@ -1,8 +1,9 @@
 @import '../../scss/variables';
+@import '../../scss/mixins';
 
 .button {
   background-color: $light-gray;
-  text-algin: center;
+  text-align: center;
```

Comment:
How was it working before or was not in use 😛

Reply:
good catch!